src/renderer/views/config/ConfigurationView.tsx

```diff
@@ -116,19 +124,18 @@ const ConfigurationView = observer(() => {
             defaultValue={electronStore.get("BlockchainStoreDirName")}
           />
           <FormLabel className={classes.newLine}>
-            <T _str="Select Language" _tags={transifexTags}/>
+            <T _str="Select Language" _tags={transifexTags} />
           </FormLabel>
-          <Select
-            name="select"
-            className={classes.selectLocale}
-            items={languages.map(({ localized_name }) => localized_name)}
-            defaultValue={
-              languages.find(({ code }) => code === selectedLocale)?.localized_name ??
-              "English"
-            }
-          />
+          {languages.length > 0 && (
+            <Select
+              name="select"
+              className={classes.selectLocale}
+              items={languages.map(({ localized_name }) => localized_name)}
+              defaultValue={selectedLanguage ?? "English"}
+            />
+          )}
```

Comment:
If `lanugages` length is 0, the configuration form is not broken? IMO the css style is might be little different.

Reply:
It seems almost okay with just the Select missing while it's fetching the languages, but it might be good idea to add indicator like a loading spinner.